src/styles/cards/card.scss

```diff
@@ -0,0 +1,113 @@
+@import "cards/content.scss";
+@import "cards/stars.scss";
+
+main {
+  padding: 50px 0;
+}
+
+.cards {
```

Comment:
looks like a separate BEM block, so should be in a separate file according to the BEM convention